components/admin_console/permission_schemes_settings/permissions_tree/permissions_tree.test.tsx

```diff
@@ -29,6 +31,7 @@ describe('components/admin_console/permission_schemes_settings/permission_tree',
         license: {
             LDAPGroups: 'true',
             isLicensed: 'true',
+            SkuShortName: LicenseSkus.Enterprise,
```

Comment:
@marianunez, I'd love your input here. I recall that doing this used to be wrong, but I'm not sure I'm up to speed on the "right" way to do things.

Reply:
Did it used to be wrong because the correct approach was to add a feature bit in the license? 

This LGTM on the new approach of relying on the sku over feature flags in the license. With the suite strategy, this is preferred to avoid bloating the license file definition and also to reduce the iterations needed to update the different repos for each new flag. More context [here](https://docs.google.com/document/d/1PhlIiGXbAl-sTRFXLRT5R71eSzUTA0HXAVNXPu1aAVU/edit#bookmark=id.u2slvcfiovn)

Reply:
Ah, I recall that document now -- thank you, @marianunez! For completeness, should I be checking against E20 as well, or have those skus already "expired" and no longer need to be checked for explicitly?

Reply:
Yes, we still support E20 licenses and there are still plenty actives in the field. However, this is more of a product decision if you want the feature available in old skus. Some features are limited to new skus. If it is intended for all high tier skus (E20 and Enterprise), then yes, you would need to add E20 to this check.